src/status_im/ui/screens/home/views.cljs

```diff
@@ -148,46 +148,47 @@
              (empty? search-filter)
              hide-home-tooltip?
              (not @search-active?))
       [welcome-blank-page]
-      [list/flat-list
-       {:key-fn                       :chat-id
-        :keyboard-should-persist-taps :always
-        :data                         chats
-        :render-fn                    render-fn
-        :header                       [:<>
-                                       (when (or (seq chats) @search-active? (seq search-filter))
-                                         [search-input-wrapper search-filter chats])
-                                       [referral-item/list-item]
-                                       (when (and (empty? chats)
-                                                  (not status-community)
-                                                  (or @search-active? (seq search-filter)))
-                                         [start-suggestion search-filter])
-                                       (when status-community
-                                         ;; We only support one community now, Status
-                                         [communities.views/status-community status-community])
-                                       (when (and status-community
-                                                  (seq chats))
-                                         [quo/separator])]
-        :footer                       (if (and (not hide-home-tooltip?) (not @search-active?))
-                                        [home-tooltip-view]
-                                        [react/view {:height 68}])}])))
+      [react/view
+       [:<>
+        (when (or (seq chats) @search-active? (seq search-filter))
+          [search-input-wrapper search-filter chats])
+        [referral-item/list-item]]
+       (when
+        (and (empty? chats)
+             (empty? communities)
+             (or @search-active? (seq search-filter)))
+         [start-suggestion search-filter])
+       (when (seq communities)
+         [communities.views/communities-home-list communities])
+       (when (and (seq communities)
+                  (seq chats))
+         [quo/separator])
+       [list/flat-list
+        {:key-fn                       :chat-id
+         :keyboard-should-persist-taps :always
+         :data                         chats
+         :render-fn                    render-fn
+         :footer                       (if (and (not hide-home-tooltip?) (not @search-active?))
+                                         [home-tooltip-view]
+                                         [react/view {:height 68}])}]])))
 
 (views/defview chats-list []
-  (views/letsubs [status-community [:communities/status-community]
+  (views/letsubs [communities [:communities/communities]
                   loading? [:chats/loading?]
                   {:keys [chats search-filter]} [:home-items]
                   {:keys [hide-home-tooltip?]} [:multiaccount]]
     [react/scroll-view
```

Comment:
for some reason scroll view still here, we need to remove it

Reply:
removed